src/components/text-editor/text-editor.tsx

```diff
@@ -82,12 +82,14 @@ export class TextEditor implements FormComponent<string> {
     public change: EventEmitter<string>;
 
     public render() {
-        return (
-            <fieldset disabled={this.readonly || this.disabled}>
+        return [
+            <span class="notched-outline">
+                <span class="leading-outline" />
                 {this.renderLabel()}
-                {this.renderEditor()}
-            </fieldset>
-        );
+                <span class="trailing-outline" />
+            </span>,
```

Comment:
if you wonder why this refactor was needed, you can see this https://github.com/Lundalogik/lime-elements/pull/2935#discussion_r1581000378